src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.ParserImpl;
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.ReaderImpl;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.WriterImpl;
+import core.basesyntax.strategy.AddOperationHandler;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Map<String, OperationHandler> operationStrategyMap = new HashMap<>();
+        operationStrategyMap.put("p", new PurchaseOperationHandler());
+        operationStrategyMap.put("s", new AddOperationHandler());
+        operationStrategyMap.put("b", new BalanceOperationHandler());
+        operationStrategyMap.put("r", new AddOperationHandler());
```

Comment:
Let's make a separate class for this

Reply:
@Dhakmar this is not a mistake if we initialize a map here
but we can indeed create a Strategy class that will choose the correct handler from the map
we may pass the map to the constructor of the Strategy class